R/formula-tools.R

```diff
@@ -512,41 +528,44 @@ construct_re <- function(re, data) {
     # and without rhs interactions
     for (k in j) {
       if (length(k) == 1) {
-          if (terms_int[i]) {
-            effects <- enw_add_pooling_effect(
-              effects, var_name = gsub(":", "__", k, fixed = TRUE),
-              finder_fn = function(effect, pattern) {
-                grepl(pattern[1], effect) &
-                  grepl(pattern[2], effect, fixed = TRUE) &
-                  lengths(
-                    regmatches(effect, gregexpr(":", effect, fixed = TRUE))
-                  ) == 1
-              },
-              pattern = strsplit(k, ":", fixed = TRUE)[[1]]
-            )
-          } else {
-            effects <- enw_add_pooling_effect(
-              effects, var_name = k,
-              finder_fn = function(effect, pattern) {
-                grepl(pattern, effect) & !grepl(":", effect, fixed = TRUE)
-              },
-              pattern = k
-            )
-          }
+        if (terms_int[i]) {
+          effects <- enw_add_pooling_effect(
+            effects,
+            var_name = gsub(":", "__", k, fixed = TRUE),
+            finder_fn = function(effect, pattern) {
+              grepl(pattern[1], effect) &
+                grepl(pattern[2], effect, fixed = TRUE) &
+                lengths(
+                  regmatches(effect, gregexpr(":", effect, fixed = TRUE))
+                ) == 1
+            },
+            pattern = strsplit(k, ":", fixed = TRUE)[[1]]
+          )
+        } else {
+          effects <- enw_add_pooling_effect(
+            effects,
+            var_name = k,
+            finder_fn = function(effect, pattern) {
+              grepl(pattern, effect) & !grepl(":", effect, fixed = TRUE)
+            },
+            pattern = k
+          )
+        }
       } else {
         if (terms_int[i]) {
           effects <- enw_add_pooling_effect(
             effects,
             var_name = paste(gsub(":", "__", k, fixed = TRUE), collapse = "__"),
             finder_fn = function(effect, pattern) {
               grepl(pattern[1], effect) & grepl(pattern[2], effect) &
-              grepl(pattern[3], effect)
+                grepl(pattern[3], effect)
```

Comment:
Does the linter want a space here?

I'm okay with a bit of side tidying along with other commits, but again: if something in the CI isn't complaining about it, please don't tweak spacing, line wraps, etc to personal taste. I am also regularly guilty of this, so totally sympathesize, but its a bad habit when working with others.

Reply:
Not done on purpose!